docs/rules/object-curly-spacing.md

```diff
@@ -145,5 +145,4 @@ You can turn this rule off if you are not concerned with the consistency of spac
 
 * [comma-spacing](comma-spacing.md)
 * [space-in-parens](space-in-parens.md)
-* [space-in-brackets](space-in-brackets.md) (deprecated)
-
+* [space-in-brackets](space-in-brackets.md)
```

Comment:
See these issues:
https://github.com/eslint/eslint/issues/2225
https://github.com/eslint/eslint/issues/2226

But maybe it's not technically deprecated until they're both completed